config.py

```diff
@@ -20,7 +20,7 @@ class Config:
     VERSION = os.getenv('VERSION', '0.1.2')
 
     SECURE_MESSAGING_DATABASE_URL = os.getenv(
-        'SECURE_MESSAGING_DATABASE_URL', 'postgresql://postgres:postgres@localhost:5432')
+        'SECURE_MESSAGING_DATABASE_URL', 'postgresql://postgres:postgres@localhost:6432')
```

Comment:
If you're changing the default database port does the README need updating to reflect this?

Reply:
Going to change this back. As this has been discussed to be part of a different PR.